rma/data/stock_data.xml

```diff
@@ -138,69 +138,4 @@
         <field name="warehouse_selectable" eval="False" />
         <field name="rma_selectable" eval="True" />
     </record>
-
-    <record id="rule_rma_customer_in_pull" model="stock.rule">
```

Comment:
Why did you remove all the rules?

Reply:
Because they were duplicated, as they were generated also by code